test/autorest/generated/azurespecialsgroup/xmsclientrequestid.go

```diff
@@ -63,7 +64,15 @@ func (client *xmsClientRequestIdoperations) getHandleResponse(resp *azcore.Respo
 
 // getHandleError handles the Get error response.
 func (client *xmsClientRequestIdoperations) getHandleError(resp *azcore.Response) error {
-	return errors.New(resp.Status)
+	msg := resp.Status
+	if resp.Body != nil {
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
```

Comment:
Probably fall back to returning `resp.Status` in this case as it's more descriptive.  We could include this error in the message though.  Maybe something like
`return fmt.Errorf("%s; failed to read response body: %w", resp.Status, err)`

Reply:
Done